setup-branch-protection.sh

```
@@ -0,0 +1,97 @@
+    "required_pull_request_reviews": {
+        "required_approving_review_count": 1,
+        "dismiss_stale_reviews": true,
+        "require_code_owner_reviews": false
+    },
+    "restrictions": null,
+    "allow_force_pushes": false,
+    "allow_deletions": false
+}'
+
+# Apply branch protection
+RESPONSE=$(gh api repos/$REPO_OWNER/$REPO_NAME/branches/$DEFAULT_BRANCH/protection \
```

Shellcheck (0.11.0)

[warning] RESPONSE appears unused. Verify use (or export if used externally).

(SC2034)